hack/install_kind.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
minor: 
```suggestion
#!/usr/bin/env bash
```

Reply:
Great additions though thanks for adding these scripts.

Reply:
I always find it annoying to bring up a devel environment asking for lots of tiny tools ':D